docs_sphinx/introduction/install.rst

```diff
@@ -24,9 +24,7 @@ Installation with Anaconda
 Installing Anaconda
 ~~~~~~~~~~~~~~~~~~~
 `Download the Anaconda distribution <https://www.anaconda.com/distribution/#download-section>`_
-for your Operating System. Note that the choice between Python 2.7 and Python
-3.x is not very important at this stage, Anaconda allows you to create a Python
-3 environment from Python 2 Anaconda and vice versa.
+for your Operating System. 
```

Comment:
Documentation should probably be updated to make it clear that there is no longer a Python 2 version, and point to the last available version for Python 2.

Reply:
Done!

Reply:
I might rewrite something like "Download the Anaconda distribution ... for your operating system. Make sure you install Python 3, as Brian is no longer available for Python 2. The last version of Brian available for Python 2 was 2.3 (?) available at ..., documentation available at ...."